Description: Configuration file fallback
 Fall back to /usr/lib/pcmciautils/config.opts if /etc/pcmcia/config.opts
 doesn't exist.
 .
 This is Debian-specific, to deal with migration from the old pcmcia-cs
 package.
Author: Colin Watson <cjwatson@debian.org>
Bug-Debian: http://bugs.debian.org/392222
Forwarded: not-needed
Last-Update: 2010-03-29

--- a/src/startup.c
+++ b/src/startup.c
@@ -186,7 +186,8 @@ static void load_config(void)
 		syslog(LOG_ERR, "chdir to %s failed: %m", configpath);
 		exit(EXIT_FAILURE);
 	}
-	parse_configfile("config.opts");
+	if (parse_configfile("config.opts") == -1)
+                parse_configfile("/usr/lib/pcmciautils/config.opts");
 	return;
 }
 
